stac_mjx/main.py

```
@@ -63,36 +69,61 @@ def run_stac(
-        fit_data = stac.fit_offsets(fit_data)
-
-        logging.info(f"saving data to {fit_offsets_path}")
-        io.save(fit_data, fit_offsets_path)
+        kps = kp_data[: cfg.stac.n_fit_frames]
+        print(f"Running fit. Mocap data shape: {kps.shape}")
+        fit_offsets_data = stac.fit_offsets(kps)
+        print(f"saving data to {fit_offsets_path}")
+        io.save_data_to_h5(
+            config=cfg, file_path=fit_offsets_path, **fit_offsets_data.as_dict()
+        )
+        (fit_offsets_data, fit_offsets_path)
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)